pkg/rancher-components/README.md

```diff
@@ -9,15 +9,78 @@ yarn
 yarn build:lib
 ```
 
-### Run your unit tests
+### Lints and fixes files
 ```
-yarn test:unit
+yarn lint
 ```
 
-### Lints and fixes files
+## Criteria for porting Dashboard components to Rancher Components
+
+We want to keep each component in Rancher Components simple and isolated to ensure that Rancher Components will be simple and easy to include across projects. We've come up with a few guidelines to help developers identify components that might be a good fit before porting:
+
+1. Is this component used in multiple projects? For example, Rancher Dashboard and Rancher Desktop maintained individual implementations of checkbox, radio button, and input components. This duplication of effort was eliminated by including these components in Rancher Components.
+2. Is this component a core building block? Core building blocks are components like buttons, inputs, labels, badges, etc.. components used to build larger and more complicated features. We can think of core building blocks akin to atoms and molecules if you subscribe to the concept of [atomic design](https://bradfrost.com/blog/post/atomic-web-design/#atoms).
+3. Is this component self-contained? A component should be able to manage its state without relying on Vuex or any API calls.
+
+### Governance
+
+Governance is the most important ingredient to creating a healthy component library that stands the test of time. We must have clear intent and understanding of usage before adding a new component to Rancher Components. Existing Dashboard components might need review before porting so that we can better define clear boundaries between Dashboard and component behavior.
+
+![Rancher Components Docs 2022-11-28 09 19 05 excalidraw](https://user-images.githubusercontent.com/835961/204343084-5b1ad6d3-b9ca-4295-a81f-3f7a02de63f6.png)
+
+## Developing and debugging
+
+Rancher Dashboard is a monorepo that houses development for several packages, including Rancher Components. We use [Yarn Workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/) to make developing in a monorepo a little easier.
+
+Running `yarn install` will automatically link inter-package dependencies for each package that has workspaces enabled. The only requirement for linking is that the version is the same in the linked package and the target library. For example, we want to make sure that the `@rancher/components` version matches the dependency for `@rancher/shell` to automatically link
```

Comment:
Since the aim is to use it also in other libraries, could you eventually say how do you develop components in case of other repositories, e.g. Rancher Desktop?